modules/graphdb/templates/01_disk_management.sh.tpl

```diff
@@ -42,8 +42,20 @@ for i in $(seq 1 6); do
   fi
 done
 
-if [ -z "$${VOLUME_ID:-}" ]; then
+# Transforms the returned result to an AVAILABLE_VOLUMES
+if [ -n "$VOLUME_ID" ]; then
```

Comment:
we can move this at the bottom and call `create_volume` in the else block instead of checking the variable again.